shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -197,18 +200,6 @@ - (void)testViewDidLoadWillInvokeCreateTouchRateCorrectionVSyncClient {
   OCMVerify([viewControllerMock createTouchRateCorrectionVSyncClientIfNeeded]);
```

Comment:
`testStartKeyboardAnimationWillInvokeSetupKeyboardAnimationVsyncClient` moved to `testFlutterViewControllerStartKeyboardAnimationWillCreateVsyncClientCorrectly`